src/app/components/AnswersWrapper.js

```diff
@@ -0,0 +1,25 @@
+import { Button } from './Button';
+
+export const AnswersWrapper = (answers, rightAnswer, chosenAnswer) => {
+  const answersWrapperObj = document.createElement('div');
+  answersWrapperObj.classList.add('answersWrapper');
+  answersWrapperObj.id = 'answersWrapper';
+
+  answers.forEach((answer, index) => {
+    const classList = ['button--answer'];
+    const isAnswerCorrect = answer === rightAnswer;
+    isAnswerCorrect
+      ? classList.push('button--correctAnswer')
```

Comment:
Jakbyśmy to tutaj zostawiali to ja jestem zawsze za jaknajmniejsza mutowalnoscia, to ulatwia kod - kiedy cos juz raz utworzone nie jest zmieniane :) Nie trzeba szukac np. "a gdzie dodalismy jeszcze jedną klasę. Można to np. w tym przypadku zrobić tak:

```
const isAnswerCorrect = answer === rightAnswer;
    const answerButton = Button({
      id: `answer${index + 1}`,
      btnText: answer,
      classList: ['button--answer', isAnswerCorrect ? 'button--correctAnswer' : 'button--incorrectAnswer'],
      onClickFn: chosenAnswer(answer, isAnswerCorrect),
    });

    answersWrapperObj.appendChild(answerButton);
 ```